plugins/woocommerce/includes/class-wc-product-simple.php

```diff
@@ -70,7 +70,7 @@ public function add_to_cart_text() {
 	 */
 	public function add_to_cart_description() {
 		/* translators: %s: Product title */
-		$text = $this->is_purchasable() && $this->is_in_stock() ? __( 'Add &ldquo;%s&rdquo; to your cart', 'woocommerce' ) : __( 'Read more about &ldquo;%s&rdquo;', 'woocommerce' );
+		$text = $this->is_purchasable() && $this->is_in_stock() ? __( 'Add to cart &ldquo;%s&rdquo;', 'woocommerce' ) : __( 'Read more about &ldquo;%s&rdquo;', 'woocommerce' );
```

Comment:
```suggestion
		$text = $this->is_purchasable() && $this->is_in_stock() ? __( 'Add to cart: &ldquo;%s&rdquo;', 'woocommerce' ) : __( 'Read more about &ldquo;%s&rdquo;', 'woocommerce' );
```

@n2erjo00 thanks for the PR! What do you think about adding a `:` between the "Add to cart" and the product name? Seems like that might help differentiate the action from the product, since "Add to cart bananas" doesn't really make sense in English.

Reply:
@coreymckrill 

I'll do some testing today and report findings with comment / commit

Reply:
All good